tests/Stormpath/Tests/Resource/ApplicationTest.php

```diff
@@ -132,6 +132,27 @@ public function testCreateAccount()
         $account->delete();
     }
 
+    public function testCreateAccountWithCustomData()
+    {
+        $application = self::$application;
+
+        $account = \Stormpath\Resource\Account::instantiate(array('givenName' => 'Account Name',
+            'surname' => 'Surname',
+            'username' => md5(time()) . 'username',
+            'email' => md5(time()) .'@unknown123.kot',
+            'password' => 'superP4ss'));
+
+        $customData = $account->customData;
+        $customData->phone = "12345";
+
+        $application->createAccount($account);
+
+        $account = \Stormpath\Resource\Account::get($account->href);
```

Comment:
Before making this call, I would create a new Client instance and get the account from there because the custom data *might* be cached (I prefer this way over the option of just creating a client with cacheing disabled).

To do this, I would set something up on the BaseTest class to create a new client with the information provided on the environment variables.